docs/source/checks/tabular/train_test_validation/plot_train_test_feature_drift.py

```diff
@@ -156,3 +157,27 @@
 #%%
 # As we see, our condition successfully detects and filters the problematic
 # features that contains a drift!
+#
+# Get an aggregated value
+# =======================
+# Using the reduce_output function we can aggregate the drift values per feature and get a collective score
```

Comment:
make the `reduce_output` a link to the api reference

Reply:
Wouldn't repeat the "aggregated" in title and first sentence
(maybe change one of the to "single" or an alternative)